DTS-STN/Alpha-Site-CMS
# Notify middleware

# Description

[Create a System Use Notification for Strapi](https://jira-dev.bdm-dev.dts-stn.com/browse/SCL-601)

This PR adds a use notification middleware to Strapi. Since session information is only stored for as long as the JWT is active, there wasn't any easy way to show past login attempts, so this middleware simply sends a message to the email that was used to login, notifying them of a login attempt.

This is accomplished by sending a POST request to GCNotify with the user's email.

## Test Instructions

1. Add required environment variables to your `.env` file (DM me for these)
2. Type `yarn develop` to start Strapi
3. Login using an email that you can access (ie. no admin@strapi.io)
4. Check your email

## Checklist


## Product and Sprint Backlog

[Jira](https://jira-dev.bdm-dev.dts-stn.com/secure/RapidBoard.jspa?rapidView=96)
